src/utils.js

```diff
@@ -64,8 +64,7 @@ export const is = {
   object: arg => typeof arg === 'object' && arg !== null,
   plainObject: arg => toString(arg) === '[object Object]',
   array: arg => Array.isArray(arg),
-  error: arg => is.object(arg) &&
-    (toString(arg) === '[object Error]' || arg instanceof Error),
+  error: arg => is.object(arg) && toString(arg) === '[object Error]',
```

Comment:
Because `toString(arg) === "[object Error]"` is redundant to `arg instanceof Error`. You can't have a object that return something different than `"[object Error]"` when using `Object.prototype.toString`